pytket/extensions/qujax/qujax_convert.py

```diff
@@ -136,53 +90,72 @@ def tk_to_qujax_args_symbolic(
     gate_name_seq = []
     qubit_inds_seq = []
     param_inds_seq = []
+    param_index = 0
     for c in circuit.get_commands():
         gate_name = c.op.type.name
         if gate_name == "Barrier":
             continue
         gate_name_seq.append(gate_name)
         qubit_inds_seq.append(_tk_qubits_to_inds(c.qubits))
-        param_inds_seq.append(
-            jnp.array([symbol_map[symbol] for symbol in c.op.free_symbols()])  # type: ignore
-        )
+        if symbol_map:
+            param_inds_seq.append(
+                jnp.array([symbol_map[symbol] for symbol in c.op.free_symbols()])  # type: ignore
+            )
+        else:
+            n_params = len(c.op.params)
+            param_inds_seq.append(jnp.arange(param_index, param_index + n_params))
+            param_index += n_params
 
     return gate_name_seq, qubit_inds_seq, param_inds_seq, circuit.n_qubits
 
 
-def tk_to_qujax_symbolic(
-    circuit: Circuit, symbol_map: Optional[dict] = None
+def tk_to_qujax(
+    circuit: Circuit, symbol_map: Union[bool, dict] = False
 ) -> qujax.UnionCallableOptionalArray:
     """
-    Converts a pytket circuit with symbolics parameters and a symbolic parameter map
-    into a function that maps circuit parameters to a statetensor.
-    Assumes all circuit gates can be found in qujax.gates.
-    Note that the behaviour of tk_to_qujax_symbolic(circuit)
-    is different to tk_to_qujax(circuit),
-    tk_to_qujax_symbolic will look for parameters in circuit.free_symbols()
-    and if there are none it will assume that none of the gates require parameters.
-    On the other hand, tk_to_qujax will work out which gates are parameterised
-    based on e.g. circuit.get_commands()[0].op.params
+    Converts a pytket circuit into a function that maps circuit parameters
+    to a statetensor. Assumes all circuit gates can be found in ``qujax.gates``
+    The ``symbol_map`` argument controls the interpretation of any symbolic parameters
+    found in ``circuit.free_symbols()``.
+
+    - If ``symbol_map`` is ``False``, circuit.free_symbols() will be ignored.
```

Comment:
What happens if there are free symbols in the case symbol_map=False ? It is not clear to me what the "will be ignored" means. If there are symbolic parameters, what happens with them?

Reply:
If `symbol_map=False`, it won't matter if there are symbolic parameters or numeric parameters.

> Parameterised gates will be determined based on whether they are stored as
> functions (parameterised) or arrays (unparameterised) in qujax.gates. The order
> of the qujax circuit parameters is the same as in circuit.get_commands().

Reply:
have simplified - removed `symbol_map=True` functionality.

I hope this is clearer now 😅